experiments/mp-game-6/draw.creatures.js

```diff
@@ -6,29 +6,48 @@
 //       multiplayer game 6.
 // --------------------------------------------------------------------
 
-function drawBoxTable(numButtons, numLights, train, roundProps) {
-    console.log("drawBoxTable has been reached")
-    var lights = ""
+function drawBoxTable(numButtons, numLights, numStars, train, roundProps) {
+    var lights = []
     for(var i = 1; i<=numLights; i++){
-        lights = lights + "<div id='light" + i + "' style='width:125px;padding:50px 0px;background-color:teal; margin:10px;" +
-                      " opacity:0.5;text-align:center;font-size:25px;border-radius:50%;display:inline-block'>light " + i + "</div>";
-        console.log("<div id='light" + i + "' style='width:125px;padding:50px 0px;background-color:teal; margin:50px;" +
-                    " opacity:0.5;text-align:center;font-size:25;border-radius:50%'>light " + i + "</div>")
+        let ran_Num = Math.floor(Math.random()*lights.length)
+        var light = "<div id='light" + i + "' style='width:125px;padding:50px 0px;background-color:" + color(i) +"; margin:10px;" +
+                " opacity:0.5;text-align:center;font-size:25px;border-radius:50%;display:inline-block'>light " + i + "</div>";
+        lights.splice(ran_Num, 0, light)
     }
-    var buttons = ""
+    var buttons = []
     for(var j = 1; j<=numButtons; j++){
-        buttons = buttons + "<div id='button" + j + "' style='width:125px;padding:50px 0px;background-color:lightgreen; margin:10px;" +
+        let ran_Num = Math.floor(Math.random()*buttons.length)
+        var button = "<div id='button" + j + "' style='width:125px;padding:50px 0px;background-color:" + color(j) +"; margin:10px;" +
                 " opacity:0.5;text-align:center;font-size:25px;display:inline-block'>button "+ j +"</div>"
-        drawTrainingScenario(roundProps, j)
+        buttons.splice(ran_Num, 0, button)
     }
+    var stars = []
+    for(var j = 1; j<=numStars; j++){
+        let ran_Num = Math.floor(Math.random()*stars.length)
+        var star = "<div id='star" + j + "' style='width:125px;padding:50px 0px;background-color:" + color(j) +"; margin:10px;" +
+                "text-align:center;font-size:25px;border-radius:20%;display:inline-block'></div>"
+        stars.splice(ran_Num, 0, star)
+    }
+    let starsStr = stars.toString()
+    starsStr = starsStr.replace(/,/g, "")
+    let buttonsStr = buttons.toString()
```

Comment:
Prefer the `Array.prototype.join` method here - safer, built in, one line

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/join